.devcontainer/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:24.10
```

Comment:
Could you split this into a separate PR if you think it could be useful to others?

Reply:
This actually got in by mistake while resolving a merge conflict. A dev container likely would be useful for contributors, but I'm not ready to make a push for it just yet (especially as building from `Dockerfile` is the slowest way to use dev containers).

I'll get this removed.